dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/csv/DefaultCsvImportService.java

```diff
@@ -96,11 +96,15 @@
     // -------------------------------------------------------------------------
 
     @Override
-    public Metadata fromCsv( InputStream input, CsvImportClass importClass )
+    public Metadata fromCsv( InputStream input, CsvImportClass importClass, boolean firstRowIsHeader )
```

Comment:
I am wondering if it is better to have a class CsvImportOptions or similar, where we encapsulate all import options for CSV. We know that there will be more import options later, and the benefit then is that we don't have to change the method signatures in classes and tests all over all the time. Sounds like a good idea?